sdk/cosmosdb/cosmos/package.json

```
@@ -108,6 +108,10 @@
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^2.0.1",
     "@azure/logger": "^1.0.0",
+    "@azure/core-tracing": "1.0.0",
+    "@azure/opentelemetry-instrumentation-azure-sdk": "^1.0.0-beta.1",
```

Definitely want core-tracing to be a range

```suggestion
    "@azure/core-tracing": "^1.0.0",
```

---

Yes, we will have default diagnostics for our database, container and query APIs.